import-service/src/importFileParser.js

```diff
@@ -0,0 +1,57 @@
+const AWS = require("aws-sdk");
+const csv = require("csv-parser");
+
+const BUCKET_NAME = "import-product-service";
```

Comment:
such things as bucket name and region are used in both lambdas, so it's better to move it to general constants)